.github/workflows/windows.yml

```
@@ -0,0 +1,83 @@
+      - library 
+  pull_request:
+    branches: 
+      - main
+
+jobs:
+  build:
+    runs-on: windows-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)